backend/apps/schemes/migrations/0008_fix_duplicate_plan_names.py

```
@@ -0,0 +1,59 @@
+# Generated by Django 5.1.4 on 2025-10-31 13:08
+
+from django.db import migrations
+from collections import Counter
+
+
+def fix_duplicate_plan_names(apps, schema_editor):
```

Ruff (0.14.3)

Unused function argument: `schema_editor`

(ARG001)